### PR TITLE
[CLOUD-2379] Allow override LAUNCH_DIR

### PR DESCRIPTION
**Pull request by ruromero**
Part of the fix for https://issues.jboss.org/browse/RHBA-530
Fixes https://issues.jboss.org/browse/CLOUD-2379
Allows the possibility for non-eap images to decide where to copy the scripts

### CODE REVIEW

**Comment by ruromero** [comment]
I have removed the user tag because for doing a `chown jboss:root` it must be root

File: os-logging/module.yaml [hunk 1]
```diff
@@ -4,4 +4,3 @@ version: '1.0'
 description: os-logging script package.
 execute:
 - script: configure.sh
-  user: '185'
```
Comment (rcernich) [inline os-logging/module.yaml]: Why was this removed?

Comment (ruromero) [reply os-logging/module.yaml]: @rcernich because this script must be run as `root` as it executes `chown` command

Comment (rcernich) [reply os-logging/module.yaml]: It was doing that before.  Is this because LAUNCH_DIR is not owned by jboss user?

Comment (ruromero) [reply os-logging/module.yaml]: @rcernich I don't know how come it works for other dependencies but `chown jboss:root <file>` can't work without root permissions UNLESS the ownership was already `jboss:root` meaning that nothing was changed
